tests/conftest.py

```diff
@@ -1,243 +1,257 @@
 import os
 from datetime import datetime
-from decimal import Decimal as PyDecimal
-import pytest
-from openpyxl import load_workbook
+from decimal import Decimal, DecimalException
+import pytest 
 from stock_indicators.indicators.common import Quote
+from dateutil.parser import parse, ParserError
```

Comment:
[Codacy](https://app.codacy.com/gh/DaveSkender/Stock.Indicators.Python/pullRequest?prid=11519902) is saying this `ParseError` is not required.  Recommend reviewing other new finding there as well.

Reply:
ok used ParserError instead of len() to check for empty string. also resolves ```Do not use `len(SEQUENCE)` to determine if a sequence is empty```